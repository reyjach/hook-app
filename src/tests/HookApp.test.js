import { shallow } from "enzyme"
import { HookApp } from "../HookApp"

describe('prueba del componente KookApp', () => { 

    test('Debe de mostrat el componente KookApp', () => { 

        const wrapper = shallow(<HookApp />)

        expect( wrapper ).toMatchSnapshot();
     })
 })