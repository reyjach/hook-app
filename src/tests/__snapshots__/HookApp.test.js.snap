// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba del componente KookApp Debe de mostrat el componente KookApp 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
